wiki/Tournaments/SSSS/3/en.md

```diff
@@ -0,0 +1,674 @@
+   - In the Grand Final, the winner of the upper bracket will be given a 2-points advantage against the winner of the lower bracket. **There will be no bracket reset in the tournament**.
+4. Players who can compete in the next round are determined by:
+   - In the the Round of 16, the Quarterfinals, and the Semifinals, each team needs to win 7 points in order to win a match. (Best of 13)
+   - In both of the Finals weeks, excluding the Grand Final match, each team needs to win 8 points in order to win a match. (Best of 15)
+   - In the Grand Final match, each player needs to win 9 points in order to win a match (Best of 17) with the winner of the upper bracket given a 2-points advantage at the start of the match.
+   - Whether there are teams who are declared to win the match by default.
+   - Whether there are teams who are disqualified from the tournament.
+
+### Match regulations
+
+1. Each team captain must use the `!roll` command once in the multiplayer lobby in order to determine the banning and picking order.
+   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
+   - The loser of the `!roll` gets the opposite by default.
+2. Each team has to ban **one beatmap** prior to the start of the match and **one other beatmap** prior to the third round of picks (that are yet to be banned or picked at that point) from the corresponding mappool. These beatmaps will not be allowed to be picked by any player during the entire match.
+3. For each map, each team is required to field either **1 Seed B player and 1 Seed C player** or **2 Seed C players** at minimum.
+   - If a team is missing or otherwise not able to ready their lower seed players for the match, they are still allowed to continue playing with one player disadvantage (2v3) or two players disadvantage (1v3) should they wish to.
+4. **There will be no warm-up beatmaps to be played in the multiplayer lobby.** Players and teams who are looking to warm up before the match are expected to do so by their own before the match commences.
+5. In a FreeMod pick, each team has to apply at least two unique mod combinations to play the beatmap with. Allowed mods are either NoMod or Easy, HardRock, Hidden, Flashlight, or any possible combinations of the four.
+   - NoMod is treated as a mod combination of its own, in which NoMod scores will be mutiplied by `x0.7`.
+   - For balancing reasons, HardRock along with Hidden+HardRock as well as Easy along with Easy+Hidden are not considered as an unique combination.
+6. The tiebreaking system works as follows:
+   - As the mappool for each stage is tailored so that the tiebreak occurs when there's exactly three maps left available to pick, during the tiebreaker, both team captains will be discreetly asked by the referee to ban one of the three remaining beatmaps from the mappool. The one beatmap that ends up not being banned will be played as the tiebreaker. 
```

Comment:
```suggestion
   - As the mappool for each stage is tailored so that the tiebreak occurs when there is exactly three maps left available to pick, during the tiebreaker, both team captains will be discreetly asked by the referee to ban one of the three remaining beatmaps from the mappool. The one beatmap that ends up not being banned will be played as the tiebreaker. 
```